src/Resources/config/services.xml

```diff
@@ -6,236 +6,34 @@
         </service>
-        <service id="Adyen\Shopware\Handlers\PaymentResponseHandlerResult" autowire="true"/>
-        <service id="Adyen\Shopware\Storefront\Controller\AdyenAccountOrderController">
-            <argument type="service" id="Shopware\Core\System\SalesChannel\SalesChannel\ContextSwitchRoute"/>
-            <tag name="controller.service_arguments"/>
-        </service>
-
-        <!--Scheduled Tasks-->
-        <service id="Adyen\Shopware\ScheduledTask\ProcessNotifications">
-            <tag name="shopware.scheduled.task"/>
-        </service>
-        <service id="Adyen\Shopware\ScheduledTask\ProcessNotificationsHandler">
-            <argument type="service" id="scheduled_task.repository"/>
-            <argument type="service" id="Adyen\Shopware\Service\NotificationService"/>
-            <argument type="service" id="Adyen\Shopware\Service\Repository\OrderRepository"/>
-            <argument type="service" id="Shopware\Core\Checkout\Order\Aggregate\OrderTransaction\OrderTransactionStateHandler"/>
-            <argument type="service" id="payment_method.repository"/>
-            <argument type="service" id="Shopware\Core\Framework\Plugin\Util\PluginIdProvider"/>
-            <tag name="messenger.message_handler"/>
-            <call method="setLogger">
-                <argument key="$logger" type="service" id="monolog.logger.adyen_notification"/>
-            </call>
-        </service>
-        <service id="Adyen\Shopware\ScheduledTask\ScheduleNotifications">
-            <tag name="shopware.scheduled.task"/>
-        </service>
-        <service id="Adyen\Shopware\ScheduledTask\ScheduleNotificationsHandler">
-            <argument type="service" id="scheduled_task.repository" />
-            <argument type="service" id="Adyen\Shopware\Service\NotificationService"/>
-            <tag name="messenger.message_handler"/>
-            <call method="setLogger">
-                <argument key="$logger" type="service" id="monolog.logger.adyen_notification"/>
-            </call>
-        </service>
-        <service id="Adyen\Shopware\ScheduledTask\FetchPaymentMethodLogos">
-            <tag name="shopware.scheduled.task"/>
-        </service>
-        <service id="Adyen\Shopware\ScheduledTask\FetchPaymentMethodLogosHandler">
-            <argument type="service" id="scheduled_task.repository"/>
-            <argument type="service" id="Adyen\Shopware\Service\ConfigurationService"/>
-            <argument type="service" id="Shopware\Core\Content\Media\MediaService"/>
-            <argument type="service" id="payment_method.repository"/>
-            <argument type="service" id="media.repository"/>
-            <argument>%shopware.media.enable_url_upload_feature%</argument>
-            <tag name="messenger.message_handler"/>
-            <call method="setLogger">
-                <argument key="$logger" type="service" id="monolog.logger.adyen_cron"/>
-            </call>
-        </service>
```

Comment:
maybe we should also have a commands.xml file, we expect some more commands to pop up in the future